### DISCUSSION TITLE
Grid Multiple column sorting

### DISCUSSION THREAD

**Discussion by ghost** [havit/Havit.Blazor#564] [Q&A] [answered]
Hi Haviteers, is it possible to sort an **HxGrid** using **multiple columns**? Thanks.

**Comment by hakenr** [comment, marked as answer]
Yes, the grid automatically tracks the history of columns selected for sorting and passes it to the `DataProvider`.
The intention is (when the `DataProvider` is implemented to use all the `SortItems` coming):
1. Click Column1, the grid is sorted by Column1.
2. Click Column2, the grid is sorted by Column2 + Column1 (for items with same order within Column2)
3. etc.

**Reply by ghost** [reply]
Thanks a bunch
